docs/cluster-guide.md

```diff
@@ -0,0 +1,108 @@
+# Guide to setting up a 3-node cluster
+
+This quick start guide provides a step-by-step process for setting up a 3-node EventStoreDB cluster, including environment preparation, certificate generation, and cluster configuration. For additional help, use our [setup wizard](https://configurator.eventstore.com/) for customized installation assistance or refer to our video tutorial on [how to get started with EventStoreDB on Windows](https://www.eventstore.com/blog/getting-started-with-eventstoredb-our-how-to-guide), or [Linux](https://developers.eventstore.com/server/v23.10/installation.html#linux). 
+
+
+
+## Preparing the environment
+
+1. Create a folder named 'Cluster'. Within it, create three subfolders: 'Node1', 'Node2', and 'Node3'.
+2. In each node folder, add a configuration file (either .yml or .txt).
+3. Within each node folder, create the following subfolders:
+    - 'Certificates' to store the certificate and private key. 
```

Comment:
all example file paths below show lowercase `certificates`. You should either use lowercase here or uppercase in the examples.